### PR TITLE
getArticleById completed

### PR DESCRIPTION
**Pull request by Superjim**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -75,4 +74,61 @@ describe("nc-news", () => {
...
+          expect(article).toHaveProperty("article_img_url");
+        });
+    });
+    test("responds with the correct object", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then((response) => {
+          const article = response.body.article;
+          expect(article.author).toBe("butter_bridge");
```
Comment (two-triangles) [inline __tests__/app.test.js]: You could have combined this and the above test. .toHaveProperty() can take a second value, where you can pass in the value/type assertion that you expect to find at that property.

Comment (Superjim) [reply __tests__/app.test.js]: expect(article).toHaveProperty("author", "butter_bridge");

like this? cheers

File: __tests__/app.test.js [hunk 2]
```diff
@@ -75,4 +74,61 @@ describe("nc-news", () => {
...
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then((response) => {
+          const article = response.body.article;
+          expect(typeof article === "object").toBe(true);
+          expect(article !== null).toBe(true);
+        });
+    });
+    test("responds with an object with the correct proptypes", () => {
```
Comment (two-triangles) [inline __tests__/app.test.js:91]: did you mean properties?

**Comment by two-triangles** [comment]
Added some comments. You will need to add some error handling now 

400 - invalid id_number type
404 - valid but none-existent id_number

**Pull request closed without merging**